packages/core/application/core-application-browser/src/app_leave.ts

```diff
@@ -17,9 +17,6 @@ import type { EuiButtonColor } from '@elastic/eui';
  *
  * See {@link AppMountParameters} for detailed usage examples.
  *
- * @public
- * @deprecated {@link AppMountParameters.onAppLeave} has been deprecated in favor of {@link ScopedHistory.block}
```

Comment:
nit: should we still make a statement that `ScopedHistory.block` is preferred in these remarks?
Also, do we want to keep `@public`?